addon/services/tour.js

```diff
@@ -449,16 +449,19 @@ export default Service.extend(Evented, {
       const currentStep = tour.steps[index];
 
       currentStep.on('before-show', () => {
-        if (this.get('modal')) {
-          const currentElement = this.getElementForStep(currentStep);
-
-          if (currentElement) {
+        const currentElement = this.getElementForStep(currentStep);
+        if (currentElement) {
+          if (this.get('modal')) {
             currentElement.style.pointerEvents = 'none';
             if (currentStep.options.copyStyles) {
               this.createHighlightOverlay(currentStep);
             } else {
               this.popoutElement(currentStep);
             }
+          } else {
```

Comment:
This could also be written:
```
} else if (currentStep.options.highlightClass) {
  $(currentElement).addClass(step.options.highlightClass);
}
```
I opted to structure it like this since exclusivity from `this.get('modal') == true` is not necessarily mutual with presence of `highlightClass`, and it's conceivable that logic could be added to the unconditional `else` in the future that's independent of the logic that conditionally requires `highlightClass`.